simplq/src/components/common/Button/button.module.scss

```diff
@@ -28,8 +32,10 @@
   }
 }
 
-.standard-button img {
-  height: 1.5rem;
+.icon {
+  display: inline;
+  vertical-align: middle;
+  margin-right: 0.5rem;
 }
 
```

Comment:
Could the icon be made horizontally centered?  Compared to the live site, it's placed a little higher up relative to the text.

Reply:
ah, the Refresh button has the issue, while the copy button looks fine. I am guessing it's because of presence of characters with 'p' and 'y' with a bellow the line stoke. Let me check.

![image](https://user-images.githubusercontent.com/11256376/102698186-8674e200-4261-11eb-923a-dfe687665c86.png)

Reply:
Does it look fine now?
![image](https://user-images.githubusercontent.com/11256376/102698432-1ff0c380-4263-11eb-8cd4-04c67bc09f02.png)

If yes, I think it has to do with the icons, not sure how to fix.

Reply:
did you mean vertical or horizontal? Can you give a screenshot?

Reply:
My bad, I meant vertical. But it's as you've shown. I think it can be fixed though.

With these changes, it looks like this.
![Screenshot from 2020-12-20 01-55-14](https://user-images.githubusercontent.com/12583528/102698872-ad81e280-4266-11eb-8b36-441007d964f9.png)


But, on the main site, it is centered.
![Screenshot from 2020-12-20 01-55-19](https://user-images.githubusercontent.com/12583528/102698875-afe43c80-4266-11eb-9751-2964332dd16f.png)

So I thought it might be a spacing issue.

Reply:
That was a easy fix, after a good night's sleep. Done